### DISCUSSION TITLE
Cannot connect my G7 to the app

### DISCUSSION THREAD

**Discussion by Dhasheider** [NightscoutFoundation/xDrip#3601] [Q&A]
I cannot get the app to get reading from my dexcom. Please see attached for errors. Any help is much appreciated!

![Screenshot_20240730_104448_xDrip+](https://github.com/user-attachments/assets/9f37f1c7-3988-4fe3-a310-1cad48f177a9)

**Comment by Navid200** [comment]
Missing QR code means that you have not scanned the required QR code.

Have a look at every step on this page:
https://navid200.github.io/xDrip/docs/Dexcom/G7.html

**Reply by Dhasheider** [reply]
Thank you! I couldn't find wether that was crucial or not. I no longer have the applicator that has the QR code on it, do I just have to wait until I put on a new sensor to connect?

**Reply by Navid200** [reply]
The QR code you need to scan is shown on the page I linked.
It's not on the applicator.
It's step 4.

**Reply by Dhasheider** [reply]
You're a godsend thank you so much! I found that page at one point but didn't have my laptop to bring it up on and I couldn't find it again after getting to a computer
